### ISSUE TITLE
Alpha 1.0.4 (Suggestion/Bug) - Unmodified fields not ignored when submitting changes

### ISSUE THREAD

**Issue by msd-rjoachim** [iamtheyammer/gam-cros-win-wrapper#19]
![image](https://user-images.githubusercontent.com/33043645/57245164-05ff6000-6fef-11e9-8dd3-7ec02e725f89.png)
In this example, "Location" was already empty. I didn't make any changes to the object before hitting "Apply Changes".

The same behavior happens if the field is modified, and applies to the "Note" field as well.

**Comment by msd-rjoachim** [comment]
Noticed a difference in behavior after modifying a field (and then staying within that asset for more modifications). Steps to reproduce - 

1. Modify an existing field (in this case, I changed the Location to "Test") and hit "Apply Changes".
The above prompts show here.

2. Without changing assets or refreshing the device manually, modify another field (I emptied "Location" and changed "User to "Test").
No confirmation prompts, either for setting Location to empty or for changing the User field.

Looks like the confirmation prompts might only show on the first initialization of an asset.
